src/typesv2.ts

```diff
@@ -0,0 +1,77 @@
+import type { Id } from './idv2.js';
+
+type OmitStrict<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+export type ValueType = 'TEXT' | 'NUMBER' | 'CHECKBOX' | 'URL' | 'TIME' | 'POINT';
+
+export type ValueOptions = {
+  format?: string;
+  unit?: string;
+};
+
+export type Value = {
+  value: string;
+  options?: ValueOptions;
+};
+
+export type Entity = {
+  id: string;
+  types: string[];
+  properties: Record<string, Value>;
+};
+
+export type Relation = {
+  id: string;
+  type: string;
+  fromEntity: string;
+  toEntity: string;
+  index?: string | undefined;
```

Comment:
The interface for this has changed in the new spec. There's additional fields like `toSpace`, `fromProperty` etc. Should double check the protobuf